spec/setup_acceptance_node.pp

```diff
@@ -1,3 +1,11 @@
 class { 'foreman::repo':
   repo => 'nightly',
 }
+
+if $facts['os']['name'] == 'CentOS' {
+  if $facts['os']['release']['major'] == 7 {
+    package { 'centos-release-scl':
+      ensure => present,
+    }
+  }
+}
```

Comment:
This isn't needed, `foreman::repo` does this.

Reply:
Got it, thanks!